### PR TITLE
minor changes and addition of unassigned devices

### PR DESCRIPTION
**Pull request by zekerpixels**
Some minor change and
Initial translation of unassigned devices
Note: The help section of unassigneddevices.txt is not yet translated

### CODE REVIEW

File: settings.txt [hunk 1]
```diff
@@ -315,7 +315,7 @@ Notification entity=Meldingen entiteit
 Notification Settings=Meldingsinstellingen
 Notifications display=Meldingen tonen
 NTP server=NTP server
-Number format=Nummerformaat
+Number format=Nummerfor
```
Comment (zekerpixels) [inline settings.txt]: user error by mistake, already corrected

File: settings.txt [hunk 2]
```diff
@@ -431,7 +431,7 @@ Text=Tekst
 Third week=Derde week
 This is required to create VMs in KVM=Dit is nodig om VMs aan te maken in KVM
 this mode requires a network switch with proper setup and support=deze modus vergt een netwerk switch met de juiste ondersteuning
-Time format=Tijdformaat
+Time format=Tijdfor
```
Comment (zekerpixels) [inline settings.txt]: user error by mistake, already corrected

File: dashboard.txt [hunk 3]
```diff
@@ -53,7 +53,7 @@ invalid devices=ongeldige opslagen
 Last check completed on %s=Laatste controle is voltooid op **%s**
 Last check incomplete on %s=Laatste controle is onvoltooid op **%s**
 Last checked on %s=Laatste controle op **%s**
-Maximum size=Maximum maat
+Maximum size=Maximum grootte
```
Comment (zekerpixels) [inline dashboard.txt:56]: correction of user error

File: dashboard.txt [hunk 4]
```diff
@@ -96,7 +96,7 @@ Transmit counters=Verzendtellers
 Transmitted packets=Verzonden pakketten
 UPS Load=UPS lading
 UPS status=UPS status
-Usable size=Bruikbare maat
+Usable size=Bruikbare grootte
```
Comment (zekerpixels) [inline dashboard.txt:99]: correction of user error

File: settings.txt [hunk 5]
```diff
@@ -315,7 +315,7 @@ Notification entity=Meldingen entiteit
 Notification Settings=Meldingsinstellingen
 Notifications display=Meldingen tonen
 NTP server=NTP server
-Number format=Nummerfor
+Number format=Nummerformaat
```
Comment (zekerpixels) [inline settings.txt]: correction of mistake

File: settings.txt [hunk 6]
```diff
@@ -431,7 +431,7 @@ Text=Tekst
 Third week=Derde week
 This is required to create VMs in KVM=Dit is nodig om VMs aan te maken in KVM
 this mode requires a network switch with proper setup and support=deze modus vergt een netwerk switch met de juiste ondersteuning
-Time format=Tijdfor
+Time format=Tijdformaat
```
Comment (zekerpixels) [inline settings.txt]: correction of user error

**Comment by bergware** [comment]
Ok to merge

**Comment by spencerjunraid** [comment]
Thanks Ronald

**Pull request merged**